### ISSUE TITLE
[Request] Items Available In Shop Option

### ISSUE THREAD

**Issue by PlantBeatdown** [Brionjv/Tomodachi-Life-Save-Editor#38]
I noticed that only the Streetpass/Spotpass items have an option to be available in the shop. Since there's no way to edit which items you have (IE: how many of an outfit you have, which interiors you have, etc.) in more specific way, since you just have the option to put X amount of the entire category (IE: Clothing, interiors, hats, etc.) into your inventory. An option to have all the items of a category be available in shop would be very nice so you can pick and choose.

**Comment by Brionjv** [comment]
Yes I need to list all items in the game for each version, it will take times.
